providers/edge/src/airflow/providers/edge/models/edge_worker.py

```diff
@@ -52,6 +52,14 @@ class EdgeWorkerState(str, Enum):
     """Edge Worker was shut down."""
     UNKNOWN = "unknown"
     """No heartbeat signal from worker for some time, Edge Worker probably down."""
+    MAINTENANCE_REQUEST = "maintenance request"
+    """Maintenance mode was requested by user."""
+    MAINTENANCE_PENDING = "maintenance pending"
+    """Edge worker received the request for maintenance, waiting for jobs to finish."""
```

Comment:
Also propose a bit of re-phrasing.
```suggestion
    """Edge worker received the request for maintenance, waiting for jobs to finish. Once jobs are finished will move to 'maintenance mode'."""
```